site/src/pages/docs/index.tsx

```diff
@@ -42,7 +42,7 @@ const DocPage = ({ doc, data }) => {
         <Box maxWidth="xl" width='100%' marginX='auto'>
           <MDXRemote {...doc} data={data} components={mdxComponents} />
 
-          { nextPage.map((page) => (
+          { nextPage.map((page, i) => (
```

Comment:
> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state.

https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=We%20don%E2%80%99t%20recommend%20using%20indexes%20for%20keys%20if%20the%20order%20of%20items%20may%20change.

Reply:
you absolutely right